client/src/main/scala/skuber/api/package.scala

```diff
@@ -556,6 +556,32 @@ package object client {
        } yield ()
      }
 
+     def deleteCollection[L <: ListResource[_]]()(
```

Comment:
Although `deleteCollection` is a logical naming choice when considering the Kubernetes REST API reference, I would prefer to use the name `deleteAll` for the method as this is consistent with the Skuber API method naming for the `watchAll` methods.
Ditto with the other new methods.